deploy_project.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -e
+
+# Function to display error message and exit
+error_exit() {
+    echo "$1" >&2
+    exit 1
+}
+
+# Build the release bundle
+echo "Building the release bundle..."
+BUILD_OUTPUT=$(xcodebuild -workspace EmpowerPlant.xcworkspace -scheme EmpowerPlant -configuration Release clean build CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO 2>&1)
+
+# Extract the first path to the .app from the build output
+APP_PATH=$(echo "$BUILD_OUTPUT" | grep -Eo '/[^ ]+\.app($| )' | sed 's/ $//' | head -n 1)
+ZIP_PATH="EmpowerPlant_release.zip"
+
+# Check if the .app path was extracted successfully
+if [ ! -d "$APP_PATH" ]; then
+    error_exit "Failed to find .app path in the xcodebuild output."
+fi
+
+# Zip the .app file
+echo "Zipping up the .app file to $ZIP_PATH..."
+zip -r "$ZIP_PATH" "$APP_PATH"
+
+# Check if gh is installed
+if ! command -v gh &> /dev/null; then
+    echo "gh is not installed, installing now..."
+    
+    # Install gh via Homebrew for macOS
+    if ! command -v brew &> /dev/null; then
+        # Install Homebrew if it's not installed
+        echo "Homebrew not found. Installing Homebrew..."
+        /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)" || error_exit "Failed to install Homebrew."
+    fi
+
+    brew install gh || error_exit "Failed to install gh."
+fi
```

Comment:
We coud remove this section and just add `gh` to the Brewfile in [my PR](https://github.com/sentry-demos/ios/pull/34)

Reply:
thanks!
resolved in https://github.com/sentry-demos/ios/pull/34